_data/apps.yml

```diff
@@ -1298,3 +1298,15 @@
     - "Unity"
     - "Scratch"
   icon: "gausssense.png"
+-
+  name: "Jasper"
+  description: "The GitHub Issue Reader"
+  website: "https://jasperapp.io"
+  repository: "https://github.com/jasperapp/jasper"
+  license: ""
```

Comment:
Oh oops, one more thing, you can also just remove this line rather than leave it empty.